src/modules/autoDecomposeDeletionProposedRelationships/AutoDecomposeDeletionProposedRelationshipsModule.ts

```diff
@@ -0,0 +1,37 @@
+import { RelationshipChangedEvent, RelationshipStatus } from "@nmshd/runtime";
+import { ConnectorRuntimeModule, ConnectorRuntimeModuleConfiguration } from "../../ConnectorRuntimeModule";
+
+export interface AutoDecomposeDeletionProposedRelationshipsModuleConfiguration extends ConnectorRuntimeModuleConfiguration {}
+
+export default class AutoDecomposeDeletionProposedRelationshipsModule extends ConnectorRuntimeModule<AutoDecomposeDeletionProposedRelationshipsModuleConfiguration> {
+    private currentIdentity: string;
+
+    public init(): void {
+        // Nothing to do here
+    }
+
+    public async start(): Promise<void> {
+        const currentIdentityResult = await this.runtime.getServices().transportServices.account.getIdentityInfo();
+        this.currentIdentity = currentIdentityResult.value.address;
+
+        this.subscribeToEvent(RelationshipChangedEvent, this.handleRelationshipChanged.bind(this));
+    }
+
+    private async handleRelationshipChanged(event: RelationshipChangedEvent) {
+        if (event.data.status !== RelationshipStatus.DeletionProposed) return;
+
+        this.logger.info("Incoming 'DeletionProposed' Relationship detected.");
```

Comment:
I feel like the event is "incoming", not the relationship itself, i.e. `"Incoming 'DeletionProposed' Relationship Event detected."`.